### PR TITLE
Fix for time alignment issue

### PR DESCRIPTION
**Pull request by devgianlu**
With my poor Rust skills I should have fixed the issue faced in #310

### CODE REVIEW

File: core/src/session.rs [hunk 1]
```diff
@@ -168,8 +176,16 @@ impl Session {
...
+                let timestamp = match SystemTime::now().duration_since(UNIX_EPOCH) {
+                    Ok(dur) => dur,
+                    Err(err) => err.duration(),
+                }.as_secs() as u64;
+
+                self.0.data.write().unwrap().time_delta = server_timestamp - timestamp;
+
                 self.debug_info();
-                self.send_packet(0x49, data.as_ref().to_owned());
+                self.send_packet(0x49, vec![0, 0, 0, 0]);
```
Comment (sashahilton00) [inline core/src/session.rs]: Is there any reason for not sending an empty vec out of interest?

Comment (devgianlu) [reply core/src/session.rs]: Nope.

**Comment by sashahilton00** [comment]
Should close #310

**Comment by sashahilton00** [comment]
Closes #310

**Comment by sashahilton00** [comment]
Thanks for this, had started implementing but you beat me to it :)

**Pull request merged**